test/e2e/suite_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	jobset "sigs.k8s.io/jobset/api/v1alpha1"
+	testutils "sigs.k8s.io/jobset/pkg/util/testing"
+	//+kubebuilder:scaffold:imports
+)
+
+const (
+	timeout  = 1000 * time.Second
```

Comment:
since this is a large interval, can you change to minutes?

Reply:
Sorry this was a remnant from debugging, changed it back to 60 sec / 1 minute.